yt_dlp/extractor/ceskatelevize.py

```diff
@@ -246,7 +294,7 @@ def _real_extract(self, url):
                     'duration': duration,
                     'formats': formats,
                     'subtitles': subtitles,
-                    'is_live': is_live,
+                    'live_status': 'is_live' if is_live else 'not_live',
```

Comment:
`'is_live': is_live,` was fine

Reply:
```py
                    'duration': duration if not is_live else None,
```
?

Reply:
Point being, `is_live` is the traditional `bool` value (so `'not_live'` still means `True`), while `live_status` is its successor with pre-defined string values (owing to an oversight where enum literals were omitted in the design of Python data types).